### PR TITLE
feat: make course recommendations a part of the course celebration

### PR DESCRIPTION
**Pull request by rgraber**
mobile:
localhost_2000_course_course-v1_edX+DemoX+Demo_Course_course-end

desktop:
Screen Shot 2021-06-17 at 3 50 18 PM

### CODE REVIEW

File: src/courseware/course/course-exit/CatalogSuggestion.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ function CatalogSuggestion({ intl, variant }) {
   );
 
   return (
-    <div className="row w-100 mx-0 my-2 justify-content-center">
+    <div className="row w-100 mx-0 my-2 justify-content-center" data-testid="catalog-suggestion">
```
Comment (Vyraal1) [inline src/courseware/course/course-exit/CatalogSuggestion.jsx:36]: Why the custom HTML5 attribute? Is it something specific to how this repo handles passing props?

File: src/courseware/data/__factories__/index.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 import './courseMetadata.factory';
 import './sequenceMetadata.factory';
+import './courseRecommendations.factory';
```
Comment (Vyraal1) [inline src/courseware/data/__factories__/index.js:3]: Where do the factories get consumed/used after being defined and imported? Was curious to see what happens to them

File: src/courseware/course/course-exit/CatalogSuggestion.jsx (hunk 1 above)
Comment (rgraber) [reply src/courseware/course/course-exit/CatalogSuggestion.jsx:36]: it's specifically for testing with rosie.js. It's a pattern I took from other places in the code

File: src/courseware/data/__factories__/index.js (hunk 2 above)
Comment (rgraber) [reply src/courseware/data/__factories__/index.js:3]: Defining a factory "registers" it with the rosie.js framework. Then we can use it to make test objects for unit tests.

**Review by Vyraal1 — APPROVED** [review]

**Review by srwang — APPROVED** [review]

**Pull request merged**